src/Roadkill.Web/App_Data/NLog.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<targets>
+		<target name="FileTarget" xsi:type="File"
+			layout="[${longdate:universalTime=True}] [${level}] ${message}"
```

Comment:
@304NotModified It should capture the exception in https://github.com/roadkillwiki/roadkill/blob/e20fb10a3b6278fd161ea9fce9705f2aa3a577f0/src/Roadkill.Core/Logging/Log.cs#L117 unless there's an easier way of doing it?